app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
```

Comment:
- [ ] kindly add the associations in your user model for user's comments and likes 👍🏿 
```suggestion
  has_many :posts, foreign_key: 'author_id'
  has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'

```

Reply:
This change has been implemented well